models/articles.model.js

```diff
@@ -32,10 +32,22 @@ exports.getArticleByIdModel = (article_id) => {
 };
 
 exports.getArticleCommentsModel = (article_id) => {
-  SQL = `SELECT comment_id, body, author, votes, created_at
+  let SQL = `SELECT comment_id, body, author, votes, created_at
```

Comment:
I know this is for a previous ticket - but well done! Global variables should be avoided, especially now that you're working on a server